### DISCUSSION TITLE
Hanging when cloning repo

### DISCUSSION THREAD

**Discussion by helobinvn** [community/community#24672] [Repositories]
<p>Does anyone observe that the cloning GitHub repo started to hang around 7 hours ago? We observed many of our CI jobs failed due to this and wonder if anyone has the same problem. Because there is no incident listed in <a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a></p>

**Comment by ashneilson** [comment]
<p>We’ve been experiencing the same difficulty from NZ.</p>
<p>It seems attempts to clone repos or download tarballs of other GitHub Actions are timing out.</p>

**Comment by helobinvn** [comment]
<p>Yes, we also observed timed out during download of tarballs of Github Action as well :frowning: I’m waiting for responding from GH Support…</p>

**Comment by felixfontein** [comment, marked as answer]
<p>I think this is the same problem as <a href="https://github.community/t/read-timeouts-and-errors-from-github-urls/219615" class="inline-onebox">Read timeouts and errors from GitHub URLs</a>.</p>

**Comment by helobinvn** [comment]
<p>Yeah, this issue has been fixed and it should be connected to this incident <a href="https://www.githubstatus.com/incidents/gh0vvxtlj5v7" class="inline-onebox" rel="noopener nofollow ugc">GitHub Status - Incident with GitHub Actions</a>. Closing this topic :slight_smile:</p>
